web/static/js/prop_types.js

```diff
@@ -58,3 +58,5 @@ export const votes = PropTypes.arrayOf(PropTypes.object)
 export const ideas = PropTypes.arrayOf(idea)
 
 export const actions = PropTypes.object
+
+export const wrap = PropTypes.bool
```

Comment:
we can omit adding `wrap` to this file , as it's more for prop types that are reused throughout the app, and `wrap` is only (currently) relevant to the `UserList` component